src/components/calculator/Calculator.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import './css/calculator.css';
+import calculate from '../../logic/calculate';
+import Result from './Result';
+import Buttons from './Buttons';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: '0',
+      next: null,
+      operation: null,
+    };
+  }
+
+  handleClick = (buttonName) => {
+    const newObject = calculate(this.state, buttonName);
+    this.setState(newObject);
+  }
+
+  render() {
+    const { total, next } = this.state;
+    return (
+      <div className="calculator-wrap">
+        <div className="calculator">
+          <Result result={!total ? next : total} />
```

Comment:
- [ ] You need to make some workaround here. The logic and everything else works really great and your code is well structured and clean. But, the result display does not update the input value. This behavior is shown in the gif file I attached below. 
- To elaborate it I added 9 and 8 to get 17. I input 9 then '+' operator and 8. But it does not update the display after I input 8, yet it only updates it when I click the '=' operator. 
![chrome-capture (2)](https://user-images.githubusercontent.com/64894927/129020931-86f5e5f5-d545-46c4-b472-ec310331e30d.gif)

Reply:
Hello Henok, I got your review.
I fixed readme file, but this second stuff, I think, the problem is in calculate.js, to get a result, we need to click on =
if you put like you said 9 + 8 you can't get a result without =

what if we want to put 9 + 88, then it will count 9+8, and then we can't get our result

Reply:
I think I understand now, I fixed the problem, check now, please.